### DISCUSSION TITLE
Short version of `--state-values-set` flag

### DISCUSSION THREAD

**Discussion by JuryA** [helmfile/helmfile#925] [Ideas]
Please consider adding a shortened variant of the `state-values-set` flag because, compared to `--set`, in my opinion, despite its potential usefulness, it is not realistic. No one will (especially users) want to type such a ridiculously long monster... Who on earth came up with that and why? 🤔😪 I would appreciate any discussion on this matter! 🙂 Nevertheless, I believe this is a "harmless" request, so I hope it will be considered... Thank you in advance!

**Comment by yxxhero** [comment]
PR is welcome.
